library/alloc/src/str.rs

```diff
@@ -619,3 +637,54 @@ impl str {
 pub unsafe fn from_boxed_utf8_unchecked(v: Box<[u8]>) -> Box<str> {
     unsafe { Box::from_raw(Box::into_raw(v) as *mut str) }
 }
+
+/// Converts the bytes while the bytes are still ascii.
+/// This is performed in chunks of `16*size_of::<usize>()` which in testing
```

Comment:
This comment is wrong, there no 16.

Reply:
Good catch. I'll rebase/squash and update the comments